src/app/models/__tests__/workspace.server.model.spec.ts

```diff
@@ -192,5 +192,37 @@ describe('Workspace Model', () => {
         mongoose.Error.ValidationError,
       )
     })
+
+    describe('getWorkspaces', () => {
```

Comment:
think this ideally belongs in the previous PR but no worries, no need to rearrange now